### PR TITLE
Add common dists

### PR DESCRIPTION
**Pull request by canyon289**
WIP PR to get early feedback

### CODE REVIEW

File: Common Distributions-Distributions.ipynb [hunk 1]
```diff
@@ -0,0 +1,411 @@
+{
```
Comment (canyon289) [inline Common Distributions-Distributions.ipynb:1]: <p>I dont use these. Do either of you? If so can you add the bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (canyon289) [inline Common Distributions-Distributions.ipynb:1]: <p>I dont use this either. Do either of you? If so can you add the bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Like variance / sd.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by twiecki** [comment]
I would add one applied example for each one where it's clear what's on the x-axis.

File: Common Distributions-Distributions.ipynb (hunk 1 above)
Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: We can drop these.

Comment (canyon289) [inline Common Distributions-Distributions.ipynb:1]: <p>I dont use these. Do either of you? If so can you add the bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (canyon289) [inline Common Distributions-Distributions.ipynb:1]: <p>I dont use this either. Do either of you? If so can you add the bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Comment by twiecki** [comment]
I would add Neg Binom.

File: Common Distributions-Distributions.ipynb (hunk 1 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: > I would add one applied example for each one where it's clear what's on the x-axis.

Great idea

File: Common Distributions-Distributions.ipynb [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
```
Comment (canyon289) [inline Common Distributions-Distributions.ipynb:1]: <p>Is the interpretation here correct? I dont use this distribution all that often</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>I actually never use these interpretations of probability distributions -- I find them incredibly confusing.</p><p>To me, the NegBin is a Gamma-Poisson, i.e an overdispersed Poisson: a Poisson random variable whose rate parameter is gamma distributed. That means I parametrize it with <code>mu</code>  (Poisson rate) and <code>alpha</code>  (Gamma rate)</p>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <p>practitioners</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <p>Don't forget the semi-colon to get rid of the MPL output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <p>Why adding this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <p>0 to 1, like probabilities</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='19'/>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <p>I think there are too many here. I'd restrict to integer values of alpha and beta, as the rule of thumb is alpha=number of successes and beta=number of failures</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='20'/>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <p>As nu approaches +infinity, StudentT becomes Normal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='25'/>

Comment (AlexAndorra) [inline Common Distributions-Distributions.ipynb:1]: <ul><li>"things" repeated twice</li><li>Examples include estimating parameters ... or standard deviation</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='28'/>

**Review by AlexAndorra — CHANGES_REQUESTED** [review]
Left comments on ReviewNB @canyon289

File: Common Distributions-Distributions.ipynb (hunk 2 above)
Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I think there's a new way to set the seed now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>*needing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Return plt? Probably don't need to return anything.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>* Number of events occuring</p><p>* Like Binomial but without upper limit.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='15'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Instead could use: Number of new users signing up to your website.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Looks like inconsistent header levels.</p><p>*probabilities</p><p>Why bounded percentages? Just percentages.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='19'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Well mainly because we often have something that's symmetrically distributed around a mean, happens in nature all the time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='22'/>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>there is but for beginner stuff I think its quite confusing. At any rate we dont need it in this notebook since theres no random draws, I removed it</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Two reasons</p><p> </p><p>We use it extensively in AB Testing and distributions lessons. Adding both for completeness and to ease into this section with something people have already seen</p><p> </p><p>It's also listed in this tweet from Thomas which is where this section derives inspiration from</p><p> </p><p>https://twitter.com/twiecki/status/1419643718110437378</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>I get what youre saying but I have a question.</p><p> </p><p>How would reword your explanation for a beginner student that doesn't know what the words Poisson, rate parameter, Gamma, or overdispersed mean?</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Could you clarify this? I could be totally wrong, but I thought Neg binomial is the number of (failed+n-1) signups until N successful signups</p><p> </p><p>So if prob_signup = .2, and we want 3 people to signup, the Neg Binom would tell use how trials until 3 people sign up successfully?</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Fixed header, all should be # top level</p><p> </p><p>Bounded percentage because beta can't model percentages over 100%, like 150% growth or something like that</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Reduced to just integers</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Added</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Do we need to mention this? I think we can leave it out since were focusing on real world applications and uses, not statistical properties</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Fixed thanks</p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>Let's keep it if you think it's useful</p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>Actually I think we should also have the Exponential in there</p><p> </p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>Actually I think we should add the Poisson and Gamma in there -- very common and will ease the introduction of NegBin (i.e Gamma-Poisson) if you want to go with my interpretation (which you don't have to; I'm just sharing my own learning path).</p><p> </p><p>Poisson is often used to model the number of independ events occurring in a fixed period of time -- number of goals scored in a football game, number of minutes I'm gonna wait for the bus at my stop, etc. This number is called the Poisson rate and has to be positive.</p><p>Another way to view the Poisson is the number of successes, like Binomial, but without an obvious upper limit.</p><p> </p><p>Gamma is only defined for positive real numbers -- like the HalfNormal but more flexible, or like the Beta but not restricted to [0, 1].</p><p> </p><p>That means you can draw positive numbers from a common Gamma and use each this draw to define a new Poisson, each of which you draw samples from -- that's the NegBin (or Gamma-Poisson)</p><p> </p><p>I like that explanantion because it's just sampling and set you up for the reasoning of hierarchical models. But it's because I just can't relate to examples like "Number of failures until N number of successes" -- I find these super confusing 😅</p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>I find Thomas' example much easier when using the Gamma-Poisson (sorry for being obsessed) 😅 </p><p>Number of new users signing up to your website is Poisson distributed (number of successes without clear upper bound), but there is overdispersion , so you need more flexibility than a Poisson would allow --&gt; Gamma-Poisson</p><p> </p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>I think people will ask what nu is, because you still have to fix or infer it</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>For thomas's example the gamma poisson thought process makes much more sense agreed.</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>What if used an example thats like number of goals attempts needed to make 2 goals</p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: I like that better, although it's still convoluted (why 2 goals? 😉 )

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Well cause its an example, It could be 3 or 4. Maybe I'll say until N successful goals or something</p><p> </p>

File: Common Distributions-Distributions.ipynb [hunk 3]
```diff
@@ -0,0 +1,930 @@
...
```
Comment (canyon289) [inline Common Distributions-Distributions.ipynb:1]: <p>For Gamma Poisson, I get that mu sets the parameterization for the gamma which we then use to parametrize the Poisson dist. What does alpha parameterize? The number of draws from each poisson?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='47'/>

File: Common Distributions-Distributions.ipynb (hunk 1 above)
Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>I think we can drop this one</p>

Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>I wouldn't drop Gamma, especially as we mention it again for NegBin. Here are the bullet points Ravin:</p><ul><li>Only for positive real numbers</li><li>Like the HalfNormal but more flexible</li><li>Like the Beta but not restricted to [0, 1]</li></ul><p> </p><p> </p>

File: Common Distributions-Distributions.ipynb (hunk 3 above)
Comment (AlexAndorra) [reply Common Distributions-Distributions.ipynb:1]: <p>No no, <code>mu</code> is the Poisson rate and <code>alpha</code> is the Gamma parameter. I haven't done it, but my prediction for the plot below is that, for a fixed mu, if you increase alpha, you can get higher draws of actual Poisson rates. For a fixed alpha, if you increase mu, the draws will get shifted to the right but there will be an upper bound to them. </p><p>LMK if this is wrong or unclear. </p><p> </p><p>I think you can use the random method of the PyMC NegBin distribution, as it's also parametrized as mu and alpha.</p><p> </p><p>You can also add this to the bullet points:</p><ul><li>A Poisson random variable whose rate parameter is Gamma distributed</li></ul><p> </p>

File: Common Distributions-Distributions.ipynb (hunk 2 above)
Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: <p>Agreed, we don't need to say that it approaches the Normal but just with increasing nu the tails become smaller and smaller.</p>

Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: <p>Alex: Why? I don't really see it used in modern Bayesian modeling. I think we really want to be very restrictive in how many we add as to not overwhelm users, and we already have quite a few.</p><p> </p>

Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: <p>While that's true, if we don't even mention the Poisson, it doesn't really matter, it's just another name. These type of connections are really neat and rewarding for us but I don't think a newcomer will really appreciate them, for them, these are all just names and telling them "in PyMC and most other places it's called NegBinomial but we prefer Gamma-Poisson because of these theoretical considerations" does not add too much intuition - I think that's more advanced.</p>

Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: <blockquote>Fixed header, all should be # top level</blockquote><p>The convention is to have # be only the title. Then section headers are ## (this would be discrete and continuous) and ### for sub-section headers (this would be for the distributions).</p><p> </p><p>This actually matters e.g. when creating a TOC.</p>

File: Common Distributions-Distributions.ipynb (hunk 3 above)
Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>in a fixed number *of trials*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I would structure the example like this:</p><p> </p><p>We have 10 users visiting our website, our true conversion rate is 30%. How many successful conversions do we expect to see?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I would prefer if we used real-world examples everywhere, that's our differentiator. How about which party a random person from a country would vote for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='14'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I wouldn't even bring this one up and instead during the NegBinomial mention that sometimes people say to use a Poisson for count data, but those people are idiots that never worked with real-world data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I don't think we even need to mention the CLT here, too theoretical.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='30'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I also don't think we need this one, it's very rarely used and I don't think we use it in the course. Less is more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='42'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>I also don't think we need this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='44'/>

**Comment by twiecki** [comment]
I think we need to take a step back here and think about what we want students to take away from this. I don't think we should teach them about "here is a list of all distributions you might encounter" or "here are some interesting theoretical relationships between some of them". Instead, I think we should teach "this is the smallest subset of distributions that are actually used in practice". This is the value that we provide: we have sifted through all the textbooks and give you the inside scoop of the things that really matter and don't bother you with the stuff that doesn't, and for me that includes the Gamma, Poisson, BetaBinomial, Gamma-Poisson.

Second, our focus is on application so I think we should have the examples be applied.

Also, I had another idea: what are these distributions actually used for *in Bayesian modeling*? Only for priors or likelihoods. So I think that's what we should mention here. So e.g. Beta is often used as a prior on parameters that are percentages or probabilities and as a likelihood if you observed percentages, which is uncommon. NegBinom is almost never used as a prior but as a likelihood for count data, etc.

File: Common Distributions-Distributions.ipynb (hunk 2 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>For TOC yes but im optimizing for slides here with consistent title size</p>

File: Common Distributions-Distributions.ipynb (hunk 3 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Way better Thanks</p><p> </p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Ill just say frequentist methods then</p>

File: Common Distributions-Distributions.ipynb (hunk 2 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Not adding exponential :)</p>

File: Common Distributions-Distributions.ipynb (hunk 3 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Removed, agreed less is more</p>

File: Common Distributions-Distributions.ipynb (hunk 2 above)
Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: <p>I would use the example to number of sales made on a single day.</p>

File: Common Distributions-Distributions.ipynb (hunk 3 above)
Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Should we put "relative belief" or "plausibility" on the y-axis for all of them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>What's the example? We can just put "Did specific user sign-up".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: I like that. Will update

File: Common Distributions-Distributions.ipynb (hunk 2 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>That example wont fit well with the n,p parameterization. Are you also suggesting I change the parametrizaiton to mu gamma?</p><p> </p>

File: Common Distributions-Distributions.ipynb (hunk 3 above)
Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Done</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Updated on push to prob of rain</p>

Comment (twiecki) [reply Common Distributions-Distributions.ipynb:1]: <p>I don't think this is right. The outcome is on the x-axis, so here it would be rain yes or no and then probability is on the y-axis.</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: Fixed

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: Updated

File: Common Distributions-Distributions.ipynb [hunk 4]
```diff
@@ -0,0 +1,709 @@
...
```
Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Is it used as a prior? I think only likelihood for binary events. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>Would add more info here that when used as a prior it's often used for parameters like in a regression. Or maybe add that as an example below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (twiecki) [inline Common Distributions-Distributions.ipynb:1]: <p>used as a prior for standard deviations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='37'/>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>This ones already mentioned twice so omitting the text as I'll speak to it</p>

Comment (canyon289) [reply Common Distributions-Distributions.ipynb:1]: <p>Added</p>

**Review by twiecki — APPROVED** [review]

**Comment by canyon289** [comment]
Added to other repo and recorded. Removing here
